cocotb/share/makefiles/Makefile.pylib.Msys

```diff
@@ -29,18 +29,18 @@
 
 # All common python related rules
 
+PYTHON_BIN?=python.exe
+
 # if not explicitly set, auto-detect python dir from system path
 ifeq ($(PYTHON_DIR),)
-	PYTHON_DIR ?= $(shell dirname $(shell which python))
+    PYTHON_DIR ?= $(shell dirname $(shell which $(PYTHON_BIN)))
 endif
 
 # make sure python dir was set properly
 ifeq ($(PYTHON_DIR),)
-	$(error "Path to Python directory must be included in system path or defined in PYTHON_DIR environment variable")
+    $(error "Path to Python directory must be included in system path or defined in PYTHON_DIR environment variable")
 endif
 
-PYTHON_BIN?=$(PYTHON_DIR)/python.exe
```

Comment:
This doesn't look right any more - if someone passes `PYTHON_DIR` explicitly, they end up with `PYTHON_DIR` and `PYTHON_BIN` pointing to different installations.

Is `PYTHON_DIR` used at all after this change?

Reply:
Are you sure you looking on the right side? I think I corrected this? And it does not seems to used anywhere. Also was hesitating to just `PYTHON_DIR`.

Reply:
Sorry - to be clear, my comment is that removing this seems unsafe, since it now does the wrong thing if the user does pass `PYTHON_DIR`.

Since `PYTHON_DIR` is used in only one place (line 49), I'd be inclined to remove the ability for the user to set the variable.

Reply:
Yes. OK